src/lib/features/events/event-service.ts

```diff
@@ -105,14 +105,14 @@ export default class EventService {
     }
 
     /**
-     * @deprecated use storeUserEvent instead
+     * For events arising from user actions you should prefer to use storeUserEvents instead
```

Comment:
So this means this one is for system action? If we consider system as a user, then shouldn't we use storeUserEvents? I think we want to identify all events with a user

Reply:
Yessir. This boils down to maybe kinda sorta not having an email. Our types say it always exists. Our database disagrees.

System user is explicitly Omit<IUser, "email"> and this flow requires an email.

Hard to fix because of the db